gaia-fast-list.js

```diff
@@ -541,15 +550,12 @@ function Internal(el) {
   this.list = this.els.list;
   this.itemContainer = el;
 
-  this.injectItemsFromCache();
   this.configureTemplates();
-  this.setEmpty(!this.els.cached);
+  this.setEmpty(true);
```

Comment:
I wonder:
* if this is expensive (because of the gradients and all)
* if we should delay it (when caching is enabled) to give a chance to the cache to kick in before we need the empty background